File: Code/2020 Vietnam Southern Provincial Contest/D.py
a = int(input())
b = int(input())
if (b - a >= 9):
	print(9)
	exit()
a %= 9
b %= 9
if (b < a):
	b += 9
f = 1
for i in range(a, b + 1):
	f = f * i % 9
if (f == 0):
	f = 9
print(f)

File: Code/2020 Vietnam Southern Provincial Contest/D.cpp
#include <bits/stdc++.h>

using namespace std;
int main() {
#ifdef LOCAL
//    freopen("input.txt", "r", stdin);
#endif // LOCAL
    int a, b;
    cin >> a >> b;
    int f = 1;
    for (int i = a; i <= b; ++i)
        f = f * i % 9;
    cout << f;
    return 0;
}
